patroni/postgresql/slots.py

```diff
@@ -42,9 +46,19 @@ def compare_slots(s1: Dict[str, Any], s2: Dict[str, Any], dbid: str = 'database'
 
 
 class SlotsAdvanceThread(Thread):
+    """Daemon process :class:``Thread`` object for advancing logical replication slots on replicas.
+
+    This ensures that slot advancing queries sent to postgres do not block the main loop and cause leader lock expiry.
+
+    :ivar daemon: Enable background long-lived process thread.
```

Comment:
`daemon` is a attribute of the `Thread` class, I don't think we should add it to the docstring.
```suggestion
```